app/models/aliases/ems_container.rb

```diff
@@ -0,0 +1 @@
+::EmsContainer = ::ManageIQ::Providers::ContainerManager
```

Comment:
@Fryguy @matthewd are the `aliases` only for backward compatibility? Are we trying to get rid of them? If so it wouldn't make sense to add a new one.